Personal_Portfolio.css

```diff
@@ -5,24 +5,28 @@ nav{
     align-items: center;
   background: red;
   position: fixed;
- 
+  left:2px;
 }
```

Comment:
🔳 I think you should edit the navigation bar position and width styling because it is not well positioned and distracting.